src/Storage/Commands.Storage/File/Cmdlet/StopAzureStorageFileCopy.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Microsoft.WindowsAzure.Commands.Storage.File.Cmdlet
 {
-    [Cmdlet(VerbsLifecycle.Stop, Constants.FileCopyCmdletName)]
+    [Cmdlet(VerbsLifecycle.Stop, Constants.FileCopyCmdletName, SupportsShouldProcess = true)]
```

Comment:
What's the default ConfirmImpact? We would like to have the ConfirmImpact= low for this cmdlet.